src/HasMedia/HasMediaCollectionsTrait.php

```diff
@@ -235,13 +229,16 @@ protected function initMediaCollections() {
 	 * @throws MediaCollectionAlreadyDefined
 	 */
 	public function addMediaCollection( $name ): MediaCollection {
-		if ( $this->mediaCollections->has( $name ) ) {
-			throw new MediaCollectionAlreadyDefined;
-		}
+
+//		if ($this->mediaCollections->has($name)) {
+//
+//		    throw new MediaCollectionAlreadyDefined;
+//		}
+//        echo 'test'; V starej verzii to zbehlo 6 krat, teraz 12.
 
 		$collection = MediaCollection::create( $name );
 
-		$this->mediaCollections->put( $name, $collection );
+        $this->mediaCollections->put( $name, $collection );
```

Comment:
The formatting is not correct on this line.

Reply:
Interesting, in code it does look ok.